Makefile.in

```diff
@@ -267,6 +267,11 @@ $(build_dir)/libmodule%.a: $(build_dir)/%.o
 	@ echo Creating library $@...
 	$(quiet) $(AR) $@ $^
 
+.PHONY: git_commit
+git_commit:
+	$(quiet) VER=$(cd $srcdir && git describe --long --always)
+	$(quiet) echo "const char * const Cha_CommitID = \"$VER\";" > cha_commitid.c
```

Comment:
This produces a file with:
const char * const Cha_CommitID = "ER";

Reply:
Well, that's neat. My local version contained the required changes to make this work, but I evidently never added them to the cache to commit them. Thanks for double-checking this!